SPECS/mariner-rpm-macros/macros

```diff
@@ -198,4 +198,50 @@ end
 %_smp_mflags "-j%{_smp_build_ncpus}"
 
 # A directory for appdata metainfo.
-%_metainfodir %{_datadir}/metainfo
\ No newline at end of file
+%_metainfodir %{_datadir}/metainfo
+
+# OpenSUSE-related macros
+%_fillupdir     %{_datadir}/fillup-templates
```

Comment:
Could you move these to a separate file? Something along the lines of `macros.fillup`. It's what we did with a bunch of other macros from different packages.